### ISSUE TITLE
Roadmap to stable 1.16 LotI

### ISSUE THREAD

**Issue by Discontinuum** [Dugy/Legend_of_the_Invincibles#391]
There is some time passed, but we don't have still a stable LotI for 1.16. I wonder what should be done to reasonably enable it?

#383 seems medium-to-high priority (doesn't hreak gameplay but gets confusing overlays)

#390 medium priority, it rarely shoots and usually don't have huge consequences, although broke Calcy in KtK

Low priority gradual replacing of {FOREACH} to [foreach], probably doesn't have to be finished before the stable, the macro still works for now

Some other deprecated stuff, but a lot of was done by edwardspec, thanks to him for that (although might need some testing just in case)

Anything else?

**Comment by Dugy** [comment]
I fixed both of these bugs.

**Comment by Discontinuum** [comment]
The only potential problem reported on the forums is #396 , beside that nothing really stops the stable I guess. I played several scenarios and watched their replays just to be sure that replacement of deprecated lua didn't break anything, seems it didn't, no oos so far

**Comment by Dugy** [comment]
Since I fixed that one now, I think there is no reason not to upload it, right?

**Comment by Discontinuum** [comment]
Yes, I believe so

**Comment by Dugy** [comment]
Okay, I uploaded it. I also edited _main to make the campaign look appropriate in the overhauled 1.16 campaign listing.

**Comment by Discontinuum** [comment]
Will you release your other campaigns as well?

**Comment by Dugy** [comment]
I think I should.
